### PR TITLE
 [MAT-880] Add HARP integration to master

### PR DESCRIPTION
**Pull request by jkotanchik-SB**
- HARP login and logout
- User Verification page for account linking
- Admin UI updates to handle HARP ID
- User UI updates to make HARP provided info read-only.
- Support Page added to help users unable to link their accounts.

### CODE REVIEW

File: src/main/java/mat/client/harp/HarpUserVerificationView.java [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        password.setWidth("170px");
+        password.addStyleName("form-control");
+
+        bluePanel.add(password);
+        bluePanel.add(new SpacerWidget());
+
+        bluePanel.add(securityQuestionAnsPanel);
+        bluePanel.add(new SpacerWidget());
+
+        buttonBar.getSaveButton().setText("Link to Mat Account");
```
Comment (RohitKandimalla) [inline src/main/java/mat/client/harp/HarpUserVerificationView.java:120]: Can we change text to MAT Account ?

Comment (jkotanchik-SB) [reply src/main/java/mat/client/harp/HarpUserVerificationView.java]: Nice catch! Fixed.

File: src/main/java/mat/client/HarpSupportWidget.ui.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            <p>
+                <i class="fa fa-frown-o fa-3x" aria-hidden="true"></i>
+                <h2>We are sorry...</h2>
+            </p>
+        </div>
+        <div id="harp-support-links-section" style="text-align: left; margin: 2em;">
+            <h4>We were unable to access the Measure Authoring Tool (MAT). The following resources may be helpful:</h4>
+            <ul>
+                <li>View the <a href="https://harp.qualitynet.org/">HARP self-service</a> page</li>
+                <li>Submit a ticket via <a href="https://oncprojectracking.healthit.gov/support/login.jsp">the Office of the National Coordinator (ONC) Jira System</a> and follow the directions on the page
```
Comment (adongare) [inline src/main/java/mat/client/HarpSupportWidget.ui.xml:14]: This line is still ambiguous to me: `follow the directions on the page`. There are no direction to follow related to MAT on ONC jira. something is missing here. I don't think we need this `and follow the directions on the page`

Comment (RohitKandimalla) [reply src/main/java/mat/client/HarpSupportWidget.ui.xml:14]: We will be updating this text, People are not going to submit any ONC jira ticket, if they are New users. A new user registration form is in development (Drupal)

Comment (adongare) [reply src/main/java/mat/client/HarpSupportWidget.ui.xml:14]: I see, thanks for clarification.

Comment (jkotanchik-SB) [reply src/main/java/mat/client/HarpSupportWidget.ui.xml:14]: That's fair. We'll have to work with Kathy, and a change will be needed in both develop and master. Going to close this PR as is so we can start testing.

**Pull request merged**